utils/create-jobs.py

```diff
@@ -32,14 +32,28 @@ def parse_cmdline(machines, tests, rfs_types):
                         help='job id for link creation: URLBASE/JOB_ID')
     parser.add_argument('-i', '--jobidx', dest='job_index',  action='store',
                         help='job index for link creation: URLBASE/JOB_ID/JOB_INDEX', default='1')
+    parser.add_argument('--img-name', dest='img_name',  action='store',
+                        help="img base name (such as agl-demo-platform) - require img_ext")
+    parser.add_argument('--img-ext', dest='img_ext',  action='store',
+                        help="img extension (such as ext4.xz) - require img_name")
 
-    return parser.parse_args()
+    args = parser.parse_args()
+
+    if (not args.img_name) != (not args.img_ext):
+        parser.error("--img-name and --img-ext require one another")
+
+    return args
 
 
 def main():
     ajt = agljobtemplate.Agljobtemplate('templates')
     args = parse_cmdline(ajt.machines, ajt.tests, ajt.rfs_types)
 
+    if args.img_name:
+        img = args.img_name + "-" + args.machine + "." + args.img_ext
+    else:
```

Comment:
nitpicking-on: I would declare `img = None` at the start of the function and just do the if here nitpicking-off

It totally works like that